server/build-plugins.js

```diff
@@ -33,6 +35,47 @@ const getPackageName = name => {
   }
 };
 
+async function installRemotePackages(installPackages) {
+  // check if connected to internet
+  // if not, skip npm install
+  const EXTERNAL_URI = process.env.EXTERNAL_URI || 'npmjs.com';
+  await require('dns').lookup(EXTERNAL_URI, async err => {
```

Comment:
I know this is from an old commit, is it possible for the `dns` lookup to resolve if the entry is cached locally?

Reply:
Hmmm, so you're wondering if this would incorrectly pass if the lookup is cached on the local machine? I haven't noticed it myself when developing on the train for example and not having connection, and since this is a constant I assume it would've hit a cache in those situations if it were going to.

Reply:
Thats true, as long from your experience it hasn't been a problem then this is cool, I was just worried about a case in which it passes when it shouldn't